tests/cases/compiler/bigintWithLib.ts

```diff
@@ -0,0 +1,50 @@
+// @experimentalBigInt: true

```

Comment:
Can you add `@declaration: true to this test case so we can verify generation of declaration file as well.`
While doing that you want to also add 
```ts
const x = 12n; // should emit type 12n
const y : 12n = 12n; // should emit type 12n
let z = 12n; // should emit type bigint in declaration file
```